### ISSUE TITLE
[Bug]: Partial SIP disable prevents specific apps to run.

### ISSUE THREAD

**Issue by airman1951** [dortania/OpenCore-Legacy-Patcher#516]
### Machine Model

iMac12,1

### Application Version

Latest Build (nightly)

### Application Variant

TUI (Text User Interface)

### What versions of macOS are you seeing the problem on?

macOS 12, Monterey, macOS 11, Big Sur

### Where does this issue happen?

Other/Non-Applicable

### What is the Isssue?

The recently applied partial SIP disable does prevent some apps (i.e. cdock) to run, because the full SIP disable is required. I indicate the iMac12,1 as HW affected, but of course the issue is referring to all.
If possible, could you please provide an option in SMBIOS setting to choose between the full and partial disable of SIP?
This issue isn't present in 0.2.5.
Thanks a lot for your work and your support.

**Comment by airman1951** [comment]
![Screenshot 2021-09-27 at 17 26 13](https://user-images.githubusercontent.com/77781242/134942158-ff4afbe3-34df-4381-a47a-6f634d2f74ca.png)

**Comment by ylluminate** [comment]
Yes, others of us are also seeing this. I suspect we can modify the config.plist manually, but of course that's neither optimal nor the goal of this project.

**Comment by airman1951** [comment]
So, I found this is not a bug, but a feature.
![Screenshot 2021-09-29 at 23 46 27](https://user-images.githubusercontent.com/77781242/135353835-77c86da2-6421-4da9-9dff-eef1a3c82704.png)
Unfortunately this feature doesn't allow some apps to run.  Again, please, let the choice for a partial or full SIP disable.
Thanks.

**Comment by airman1951** [comment]
Never intended that 0.3.0 has been released.  Instead my suggestion  is stricly related to the nature of 'nightly' of this build nowadays.

**Comment by airman1951** [comment]
Well, this is definitely wrong
![Screenshot 2021-10-01 at 23 01 18](https://user-images.githubusercontent.com/77781242/135685982-9df0dbe5-6edf-446d-b315-411d123b7d8e.png)
.

**Comment by airman1951** [comment]
Useing instead 0FEF it's accepted. Let me reboot.
